src/components/schedule-for-stop/scheduleAccordionItem.svelte

```diff
@@ -0,0 +1,113 @@
+<script>
```

Comment:
the title of this file should be `ScheduleAccordionItem.svelte`, right?

Reply:
Yes